samples/excel/30-events/setting-changed.yaml

```diff
@@ -1,7 +1,7 @@
-order: 3
+order: 8
 id: excel-events-setting-changed
-name: Handle the settings-changed event
-description: This snippet shows how to register a handler for the SettingsChanged event.
+name: Workbook events
+description: Registers an event handler that runs when workbook's settings change.
```

Comment:
either add the word "the" -> `...when the workbook's settings change.`   OR remove `'s` -> `...when workbook settings change.`